subiquity/models/filesystem.py

```diff
@@ -569,7 +569,6 @@ class Disk(_Device):
     wwn = attr.ib(default=None)
     multipath = attr.ib(default=None)
     path = attr.ib(default=None)
-    model = attr.ib(default=None)
```

Comment:
Why is this line being removed?

Reply:
I replaced this with a model property because I didn't see this being set anywhere. The only reference I could find is below but I think it was always None.

https://github.com/canonical/subiquity/blob/bd4e4785a9bc4f44ceb7423bac09502719819971/subiquity/models/filesystem.py#L601